lib/explorer/data_frame.ex

```diff
@@ -1088,19 +1147,13 @@ defmodule Explorer.DataFrame do
   def distinct(df, opts) do
     opts = Keyword.validate!(opts, columns: [], keep_all?: false)
```

Comment:
The default for `columns:` must be nil. This is to avoid a similar issue as the one we detected in `drop_nil` where empty list is "ambiguous".